### PR TITLE
set user and device info header in search request

### PR DESCRIPTION
**Pull request by santosh1994**
@amarts @rahulcs @floydpraveen Please review

### CODE REVIEW

File: unbxdSearch.js [hunk 1]
```diff
@@ -1137,6 +1138,77 @@ var unbxdSearchInit = function(jQuery, Handlebars){
...
+      }
+
+      return result;
+    }
+    ,readCookie : function(name){
+      try{
+        return this.cookie('unbxd.' + name, undefined, { path: '/'});
+      }catch(e){
+        // Unbxd.log(e);
+      }
```
Comment (amarts) [inline unbxdSearch.js]: if there are possible errors here, better to log anyways, instead of consuming it. 

@rahulcs @floydpraveen anythoughts?

File: unbxdSearch.js [hunk 2]
```diff
@@ -1137,6 +1138,77 @@ var unbxdSearchInit = function(jQuery, Handlebars){
...
+            break;
+          }catch(e){}
+        }
+      }
+
+      return result;
+    }
+    ,readCookie : function(name){
+      try{
+        return this.cookie('unbxd.' + name, undefined, { path: '/'});
```
Comment (floydpraveen) [inline unbxdSearch.js]: @santosh1994 where are second and third argument used ?

File: unbxdSearch.js (hunk 1 above)
Comment (floydpraveen) [inline unbxdSearch.js]: @amarts we need to implement `Unbxd.log` method which logs when a cookie is set, till then this is fine I feel

Comment (floydpraveen) [inline unbxdSearch.js]: @santosh1994  can you add `Unbxd.log` method, like we have in tracker  ?

Comment (amarts) [inline unbxdSearch.js]: @santosh1994 raise an issue and work on that separately.. you can send another commit for that.

File: unbxdSearch.js (hunk 2 above)
Comment (santosh1994) [inline unbxdSearch.js]: second and third arguments not needed here since I modified `cookie : function (key)` for read only purpose.

**Comment by santosh1994** [comment]
@floydpraveen @amarts Made changes

**Comment by amarts** [comment]
Like this 👍

**Comment by floydpraveen** [comment]
increasing version number has missed, @santosh1994 can you send a commit for that ?
